devtools/composer.json

```diff
@@ -4,7 +4,7 @@
         "ext-json": "*",
         "ext-simplexml": "*",
         "apigen/apigen": "^2.8",
-        "fabpot/php-cs-fixer": "^1.11",
+        "friendsofphp/php-cs-fixer": "^1.11",
```

Comment:
please update directly to ^2.1 as requested in prev PR.
I'm fine with using ruleset:
```php
        '@Symfony' => true,
        '@Symfony:risky' => true,
        'array_syntax' => array('syntax' => 'long'),
        'concat_space' => array('spacing' => 'one'),
        'phpdoc_no_package' => false,
```

and apply the rules.